docs/troubleshooting.md

```diff
@@ -147,9 +147,83 @@ In some cases, the field names in filters may not reflect changes after subseque
    - Click ![](./res/icon-clear-filter.svg) to remove the current filter.  
    - Reapply the filter by clicking on the filter hyperlink, within the bot conversation, again.  
 
+> [!Note]
+> This issue has been resolved in **FortiSOAR v7.6.1** and later.
 
```

Comment:
Troubleshooting should have clear three section for SOC, PB and Connector generatio

Reply:
For the users it is one assistant and hence creating separate sections is redundancy, IMO.